api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -3209,4 +3223,24 @@ public void saveRetrospectiveOrder_shouldFailIfAnActiveDrugOrderForTheSameConcep
         expectedException.expectMessage("Cannot have more than one active order for the same orderable and care setting");
         orderService.saveRetrospectiveOrder(order, null);
     }
+	@Test
+	@Verifies(value = "should not save order if invalid orderGroup encounter", method = "saveOrder(Order)")
+	public void saveOrder_shouldNotSaveOrderIfInvalidOrderGroupEncounter() throws Exception {
```

Comment:
Since this is intended to be a unit test which tests validation logic in the OrderValidator so it belongs to OrderValidatorTest